### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by Ninjaphil24**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+> This is the professional developer portfolio of Philip Modinos.  You can visit this website at:  https://ninjaphil24.github.io/portfolio_1/ <br>Philip has been coding since 2019, focusing on html/css and php/mysql using XAMPP.  He has developed several sites for German music companies "Opera Classica Europa" and "Bad Ems Music Festival" including the site www.offenbachgp.com, which was an online singing competition, that included applause and comments by the audience and a fully automated registration system.  
 The current webpage is fully responsive and has several live links, leading to other existing projects.  For the inactive links, there will be an update later in time, once those projects have been reuploaded.  
 
 ![screenshot](./images/readmedemo.png)
```
Comment (brytebee) [inline README.md]: - [ ] **Required:** Please update your README with the live demo link, so visitors to the repo can easily see your portfolio.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Ninjaphil24👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- You have successfully deployed the page✔️
- The website works as it should✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Ninjaphil24) [reply README.md]: Hello.  It's right there.  It's being highlighted by github as an alteration of the commit.  And it works.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # Developer's Portfolio of Philip Modinos
 
-> This is the professional developer portfolio of Philip Modinos.  Philip has been coding since 2019, focusing on html/css and php/mysql using XAMPP.  He has developed several sites for German music companies "Opera Classica Europa" and "Bad Ems Music Festival" including the site www.offenbachgp.com, which was an online singing competition, that included applause and comments by the audience and a fully automated registration system.  
+> This is the professional developer portfolio of Philip Modinos.  You can visit this website at:  https://ninjaphil24.github.io/portfolio_1/ <br>Philip has been coding since 2019, focusing on html/css and php/mysql using XAMPP.  He has developed several sites for German music companies "Opera Classica Europa" and "Bad Ems Music Festival" including the site www.offenbachgp.com, which was an online singing competition, that included applause and comments by the audience and a fully automated registration system.
```
Comment (brytebee) [inline README.md]: - [ ] **Optional:** Use the **_Live Demo_** subheader as specified on the [Microverse template](https://github.com/microverseinc/readme-template) to always document links to your demo page on the README.

**Review by brytebee — APPROVED** [review]
Hi, @Ninjaphil24 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Ninjaphil24) [reply README.md]: OK, I understand now.  Apologies.  It's done.

**Pull request merged**